packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -34,10 +38,18 @@ function PostToSquadModal({
   requestMethod = request,
   ...props
 }: PostToSquadModalProps): ReactElement {
+  const notificationState = useState(true);
```

Comment:
why do you need state if it can't be set?

Reply:
It would be set from some components down the line (we pass this as a whole, didn't want to separate to two props).